ledger/participant-integration-api/src/main/scala/platform/index/BuffersUpdater.scala

```diff
@@ -38,12 +40,15 @@ import scala.util.control.NonFatal
 private[index] class BuffersUpdater(
     subscribeToTransactionLogUpdates: SubscribeToTransactionLogUpdates,
     updateCaches: (Offset, TransactionLogUpdate) => Unit,
+    metrics: Metrics,
     minBackoffStreamRestart: FiniteDuration,
     sysExitWithCode: Int => Unit,
 )(implicit mat: Materializer, loggingContext: LoggingContext, executionContext: ExecutionContext)
     extends AutoCloseable {
 
   private val logger = ContextualizedLogger.get(getClass)
+  private val cachesUpdaterExecutionContext =
+    ExecutionContext.fromExecutor(Executors.newSingleThreadExecutor())
```

Comment:
Can we name the thread in that pool for ease of log/jfr analysis?

Reply:
8dc7e0e0a4363b56205c60bfd98e04c76b73a325